klayout.dockerfile

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+
+FROM hdlc/build:build AS build
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+    g++ ruby-dev python3-dev qt5-default qt5-qmake qttools5-dev libqt5xmlpatterns5-dev qtmultimedia5-dev libqt5svg5-dev \
+ && apt-get autoclean && apt-get clean && apt-get -y autoremove \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/KLayout/klayout.git /tmp/klayout \
+ && mkdir -p /opt/klayout/usr/local/bin \
+ && cd /tmp/klayout && ./build.sh -bin /opt/klayout/usr/local/bin -rpath /usr/local/lib -option -j$(nproc) \
+ && mkdir -p /opt/klayout/usr/local/lib \
+ && cd /opt/klayout/usr/local/bin && mv libklayout* db_plugins lay_plugins ../lib/
```

Comment:
Why do we need to copy these resources manually? Are those missing from https://github.com/KLayout/klayout/blob/master/build.sh#L720-L728? Or are those optional artifacts?

Reply:
Yes, -rpath option only seems to influence the search path for libs and plugins but not install them there so I do that manually.
This may be ripe for bug report to klayout but as I am already spread too thin I leave that to other people. Same story for 'klayout -h' and 'klayout -v' being executable without X environment.